imports/ui/QuestionSidebar.jsx

```diff
@@ -199,6 +206,11 @@ export class QuestionSidebar extends ControlledForm {
             }
           </select>
           
+          <select onChange={this.setApproved} className='ql-header-button question-type form-control'>
```

Comment:
We have to be careful with the implication here. In principle, it means that I can select to show questions without approval, but those won't be published to begin with if the course has that property set to false, so it would never show anything additional. I would only show this control to "limit to approved questions" as a check box if the course allows for the possibility of unapproved questions being displayed. Maybe we should think more carefully for what the use case is for allowing unapproved questions in the course. It seems that it might only be relevant for the public library?
As a minimum, I think that this control has to change depending on whether the query of the questions is limited to approved questions or not.